cmd/veneur-emit/main.go

```diff
@@ -224,9 +242,25 @@ func destination(hostport *string, useSSF bool) (string, net.Addr, error) {
 	return addr, netAddr, nil
 }
 
+func inferTraceIDInt(id *int64, envKey string) (err error) {
```

Comment:
Why is this a pointer instead of an int? The zero value should work fine for us here (we already define `0` to be a not-valid span or root ID).